skymp5-server/cpp/lib_espm/Combiner.h

```diff
@@ -102,17 +109,23 @@ class Combiner
 
 // Helpers/utilities
 
-inline const RecordHeader* FindInGroup(const BrowserInfo& brInf,
-                                       const GroupHeader* gr,
+inline const RecordHeader* FindInGroup(const BrowserInfo& browserInfo,
+                                       const GroupHeader* group,
                                        uint32_t globFormId)
 {
+  // TODO: ideally, we should be able to get browser reference here.
+  // For now, keeping CombineBrowser. Should be changed if required.
   const RecordHeader* res = nullptr;
-  gr->ForEachRecordRecursive([&](const espm::RecordHeader* rec) {
-    if (brInf.ToGlobalId(rec->GetId()) != globFormId)
-      return false;
-    res = rec;
-    return true;
-  });
+  assert(browserInfo.parent);
```

Comment:
Will break in release build. Consider throwing an exception instead. Please also remove `#include <cassert>`.

Reply:
See https://github.com/skyrim-multiplayer/skymp/pull/110#discussion_r684787872